### ISSUE TITLE
Excessive amount of resources in Mining system structure 

### ISSUE THREAD

**Issue by Defetyre** [Aureljz/WhenDungeonsArise--Forge#10]
You can find all the diamonds, iron, gold and fully enchanted gear that you will ever need in just one tame structure I found 20 minutes into the game.
![2022-12-05_00 13 24](https://user-images.githubusercontent.com/88449275/205505392-626af1ce-a3b8-4228-acf7-a5cf0c58e6e9.png)

**Comment by Defetyre** [comment]
Wouldn't be such a big deal if there was a config file, but there is none in 1.19. So I have to remove your mod from my modpack just because of one loot table

**Comment by Insane96** [comment]
Just change the loot table

**Comment by Defetyre** [comment]
Although it can be fixed fairly easily, I don't think that this just a problem with my preferences, as the loot table obviously doesn't have intended amount of resources, there are dozens of those barrels with all have far more generous contents that even the most rare of treasure loot tables in vanilla.
Players shouldn't go out of their way to change loot tables invidiously each time they discover something like this in-game, at least not when the problem is so apparent and easy to fix.
[mining_system_barrels.txt](https://github.com/Aureljz/WhenDungeonsArise--Forge/files/10517097/mining_system_barrels.txt)

**Comment by Insane96** [comment]
Sure, but removing a mod when you can easily just change the loot is overexagereted

**Comment by Defetyre** [comment]
Yeah, but at the time I didn't realize how easy it was, so I just didn't bothered. Besides now that I remembered, it wasn't my only reason to remove the mod; when major structures loaded it caused immense lag, and every time I generated a new world there was several big structures crammed near the spawn. There were also major issues with mob spawning due to the large number of spawners that summoned mobs with gear (that don't despawn), which in turn caused natural mobs to not spawn near them. All of those combined can create some really unpleasant situations.
But all of it was somewhat bearable before I encountered this bug that outright nukes game progression. Then I just stopped caring about this mod because there could be a lot more of these surprises in store.
